File: fix.py
import re

with open('src/pages/Home.tsx', 'r') as f:
    content = f.read()

# Add a missing closing div
content = re.sub(r'        </div>\n      </section>\n    </div>\n  \);\n}', '        </div>\n        </div>\n      </section>\n    </div>\n  );\n}', content)

with open('src/pages/Home.tsx', 'w') as f:
    f.write(content)

File: fix.js
const fs = require('fs');
let text = fs.readFileSync('src/pages/Home.tsx', 'utf8');
text = text.replace('        </div>\n      </section>', '        </div>\n        </div>\n      </section>');
fs.writeFileSync('src/pages/Home.tsx', text);
